### PR TITLE
add minifill v0.0.3 with git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/thednp/minifill
**Watch 1
  Star 0
 Fork 1**
@Amomo could u help me check this PR for #8548  ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity (a denpendency of bootstrap.native)
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry 
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) 
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/minifill/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "https://github.com/thednp/minifill.git"
+  },
+  "keywords": [
+    "6k",
+    "polyfill"
+  ],
+  "author": "dnp_theme",
+  "license": "MIT",
+  "homepage": "https://github.com/thednp/minifill",
```
Comment (PeterDaveHello) [inline ajax/libs/minifill/package.json]: Remove this meaningless homepage, no need to point to repo.

**Comment by thednp** [comment]
Thanks for this too :)

**Comment by PeterDaveHello** [comment]
@thednp No problem.

**Comment by maruilian11** [comment]
@Amomo i removed `homepage`, could you help me check this PR? thanks

**Comment by thednp** [comment]
I updated [README](https://github.com/thednp/minifill/blob/master/README.md) to let users know about it too :)

**Comment by PeterDaveHello** [comment]
@thednp Thanks.

**Pull request merged**